internal/dispatch/graph/check_test.go

```diff
@@ -30,7 +29,7 @@ import (
 var ONR = tuple.ObjectAndRelation
 
 func TestSimpleCheck(t *testing.T) {
-	defer goleak.VerifyNone(t, append(testutil.GoLeakIgnores(), goleak.IgnoreCurrent())...)
```

Comment:
Was this no longer necessary?

Reply:
no - it's moved into a `TestMain` function which makes it execute for all tests in the package